sqladmin/templates/details.html

```diff
@@ -25,8 +25,10 @@ <h3 class="card-title">{{ model_admin.pk_column.name }}: {{ model_admin.get_attr
               <a href="{{ model_admin._url_for_details(value) }}">({{ value }})</a>
               {% endfor %}
               </td>
-              {% else %}
+              {% elif model_admin._has_attr(model, attr) %}
               <td><a href="{{ model_admin._url_for_details_with_attr(model, attr) }}">{{ model_admin.get_detail_value(model, attr) }}</a></td>
+              {% else %}
+              <td>{{ model_admin.get_detail_value(model, attr) }}</td>
```

Comment:
Can you please clarify why this is needed?

Reply:
Hmm. I added it because I saw the case `None` is assigned to `model` during writing the tests, but I cannot reproduce it now. :thinking: I'll cancel these changes.

Reply:
```python
=================================== FAILURES ===================================
______________________________ test_column_labels ______________________________

client = <starlette.testclient.TestClient object at 0x7f22be8b1550>

    def test_column_labels(client: TestClient) -> None:
        user = User(name="Foo")
        session.add(user)
        session.commit()
    
        response = client.get("/admin/user/list")
    
        assert response.status_code == 200
        assert response.text.count("Email") == 1
    
>       response = client.get("/admin/user/details/1")

tests/test_admin_sync.py:310: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.venv/lib/python3.8/site-packages/requests/sessions.py:600: in get
    return self.request("GET", url, **kwargs)
.venv/lib/python3.8/site-packages/starlette/testclient.py:473: in request
    return super().request(
...
.venv/lib/python3.8/site-packages/starlette/templating.py:37: in __init__
    content = template.render(context)
.venv/lib/python3.8/site-packages/jinja2/environment.py:1301: in render
    self.environment.handle_exception()
.venv/lib/python3.8/site-packages/jinja2/environment.py:936: in handle_exception
    raise rewrite_traceback_stack(source=source)
sqladmin/templates/details.html:1: in top-level template code
    {% extends "layout.html" %}
sqladmin/templates/layout.html:1: in top-level template code
    {% extends "base.html" %}
sqladmin/templates/base.html:15: in top-level template code
    {% block body %}
sqladmin/templates/layout.html:49: in block 'body'
    {% block content %} {% endblock %}
sqladmin/templates/details.html:29: in block 'content'
    <td><a href="{{ model_admin._url_for_details_with_attr(model, attr) }}">{{ model_admin.get_detail_value(model, attr) }}</a></td>
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <tests.test_admin_sync.UserAdmin object at 0x7f22bfb20940>
obj = <tests.test_admin_sync.User object at 0x7f22be8e2820>
attr = <RelationshipProperty at 0x7f22bfb72040; profile>

    def _url_for_details_with_attr(self, obj: Any, attr: RelationshipProperty) -> str:
        target = getattr(obj, attr.key)
>       pk = getattr(target, attr.mapper.primary_key[0].name)
E       AttributeError: 'NoneType' object has no attribute 'id'

sqladmin/models.py:592: AttributeError
```

Reply:
But this code seems outdated to me, this is already fixed and changed: https://github.com/aminalaee/sqladmin/blob/main/sqladmin/models.py#L593